build.sbt

```diff
@@ -1,5 +1,6 @@
 ThisBuild / organization := "io.latis-data"
 ThisBuild / scalaVersion := "2.13.5"
+reStart / mainClass := Some("latis.server.Latis3Server")
```

Comment:
I'd put this in the `settings` block below.

Reply:
That's how it looks in my local copy I just hadn't pushed it. I'll push now.

Reply:
Cool. 👍